Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM public.ecr.aws/docker/library/python:3.9.19
```

Comment:
Looking at https://gallery.ecr.aws/docker/library/python I see quite a few later python version 3.9.x

Reply:
Oh, so there are! I think I weirdly assumed that the newest version was at the top...or something? I will try using a newer version!!